common/static/common/templates/discussion/thread-show.underscore

```diff
@@ -1,21 +1,10 @@
 <div class="discussion-post">
-    <header>
-        <div class="group-visibility-label">
```

Comment:
Context: The group-visibility-label div was moved to further down in the post area. Instead of adding this to the top of the post, is has moved into the "post-context" area under the post-body area. This area also conveys when a post "relates to" a specific area of the courseware. Moving the element was done in order to clean up the post-details area, making the visual focus rest on the image / title of the post.